### ISSUE TITLE
How did you use crossentropy without sigmoid after denselayer ? 

### ISSUE THREAD

**Issue by fsalmasri** [KangBK0120/CAM#3]
How did you use crossentropy without sigmoid after denselayer ?

**Comment by KangBK0120** [comment]
Hi @fsalmasri 

Sorry for the late reply.

In Pytorch, as far as I know, Softmax function is included in the Cross Entropy loss.
Therefore, I didn't add Softmax layer in the model.
See https://discuss.pytorch.org/t/do-i-need-to-use-softmax-before-nn-crossentropyloss/16739
